.github/workflows/gh-pages.yml

```
@@ -0,0 +1,32 @@
+    branches:
+      - master # default branch
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install Pnpm
+        uses: pnpm/action-setup@v4
+        with:
```

actionlint

"with" section should not be empty. please remove this section if it's unnecessary

(syntax-check)